tests/testthat.R

```diff
@@ -1,4 +1,7 @@
 library(testthat)
 library(reticulate)
 
+py_discover_config("numpy")
```

Comment:
You can't just call `py_config` unqualified as it will fail on systems where there is no compatible version of Python (e.g. some CRAN servers). That's why we use `skip_if_no_python()`